test/conformance/kuttl/generate/validation/clusterpolicy/permissions/same-kind/README.md

```diff
@@ -0,0 +1,12 @@
+## Description
+
+This test ensures that the auth checks for generate policy is performed against given the APIVersion and the subresource.
+
+## Expected Behavior
+
+The test fails if the policy that generates `k8s.nginx.org/v1/policy` and its subresource can be created, otherwise passes.
```

Comment:
Wrong API group written here.

Reply:
Why? Kyverno should block the policy that generates `k8s.nginx.org/v1/policy` while allowing `kyverno.io/v1/policy`.

Reply:
I see, my mistake. I didn't look this over well enough.